flink-runtime/src/main/java/org/apache/flink/runtime/dispatcher/DispatcherServices.java

```diff
@@ -74,22 +78,24 @@ public DispatcherServices(
 			@Nonnull GatewayRetriever<ResourceManagerGateway> resourceManagerGatewayRetriever,
```

Comment:
I'm afraid `org.apache.flink.runtime.dispatcher.DispatcherServices` has a name conflict with `org.apache.flink.runtime.dispatcher.runner.AbstractDispatcherLeaderProcess.DispatcherService`. Technically we can distinguish by package/namespace but it possibly brings confusion later.